### ISSUE TITLE
Failure to boot DVD on 2015 15" MBP

### ISSUE THREAD

**Issue by adespoton** [ranma42/TigerOnVBox#8]
I've had this issue booting 10.4 and 10.5 since switching to a Haswell processor.  The images that booted fine on my old Core2Duo machines just drop to the UEFI shell.  So, I figured I'd use your config since it had a few settings I hadn't used before.  Same issue: dropping to shell instead of booting the DVD image.
<img width="795" alt="screen shot 2019-01-19 at 10 30 22 pm" src="https://user-images.githubusercontent.com/4846661/51436061-6df58a80-1c3a-11e9-8e90-0b1ba686619d.png">
Image is from attempting to boot a stock import under VirtualBox 6.0.2 running on OS X 10.14.2 on a MBP 15" (Mid 2015).

Log follows:
[VBox.log](https://github.com/ranma42/TigerOnVBox/files/2776335/VBox.log)

**Comment by adespoton** [comment]
Interesting... that was with 10.4.7 Client install; the Server install is working perfectly.

**Comment by ranma42** [comment]
AFAIK for 10.4 only the Server version is officially supported in virtualized environments.

**Comment by umbertocerrato** [comment]
Hello everyone, nice guide. Thank you @adespoton for sharing it in the emaculation forum. I had your same issue using Mac OS X Tiger, not the Server version.

I downloaded the Server one from Apple Developer portal and it works great. But I’m stuck in the Serial Number step.

How did you do? On the dev site there is written "See the separate download for a Developer Use Serial Number for Mac OS X Server 10.4.x" but I cannot see any "separate download"...

Best

**Comment by adespoton** [comment]
While it’s the only one officially supported due to license, there are older versions of VirtualBox that have no difficulty running 10.4 and 10.5 on older hardware.

I’m pretty sure that with enough tweaks they would run, it’s just against the license agreement which states those OSes must be run directly on actual Apple hardware.

That said, back in 2009 Apple had no issues with us running our developer copies in a VM to test software.  I don’t think they would have been so gracious with consumer copies.

**Comment by umbertocerrato** [comment]
Thank you.

But still I am not understanding which Serial Number did you entered/where you picked it from... I’m using this dmg file: "mac_os_x_server_v10.4.7_universal_build_8k1079.dmg" it should be the same used in this guide.

Maybe to obtain a SN code I have to have the Developer Membership something?

Anyway, I would be really happy to understand how this guide can be ported to QEMU, since I like it the most. And I think it can be done 99%. Also I hope on QEMU I can utilize my retail version instead of the Server.

Best

**Comment by umbertocerrato** [comment]
Solved

(Still looking for a QEMU alternative to VBOX)

**Comment by adespoton** [comment]
QEMU-PPC works quite well with 10.4 and 10.5.  Are you looking for a solution on QEMU-X86/KVM?  It should have similar settings.

**Comment by umbertocerrato** [comment]
Yes, PPC works well. I’m looking for QEMU-X86 though. I tried with many configurations but nothing.
